### ISSUE TITLE
neo4j-admin import fails when node has >20 labels

### ISSUE THREAD

**Issue by bdm83** [neo4j/neo4j#12354]
The `neo4j-admin import` tool will fail to import a CSV file when an entry has more than 20 labels. 

**neo4j-admin Version:** 3.5.8
  
### Steps to reproduce
1. Download the attached [example.txt](https://github.com/neo4j/neo4j/files/3933009/example.txt) and rename to `example.csv` 
1. Run `neo4j-admin import`:
    ```
   neo4j-admin import --database="example.db" --nodes="example.csv"
    ```
1. Verify that the import failed
1. Edit `example.csv` and remove a single label and rerun the import
1. Verify that the import is successful

### Expected behavior
The node to be imported successfully regardless of the number of labels.

### Actual behavior
The following error is reported:
```
IMPORT FAILED in 304ms.
Data statistics is not available.
Peak memory usage: 0.00 B
Error in input data
Caused by:ERROR in input
  data source: BufferedCharSeeker[source:/xxx/example.csv, position:196, line:1]
  in field: :LABEL:2
  for header: [id:ID, :LABEL]
  raw field value: label_0;label_1;label_2;label_3;label_4;label_5;label_6;label_7;label_8;label_9;label_10;label_11;label_12;label_13;label_14;label_15;label_16;label_17;label_18;label_19;label_20
  original error: null

WARNING Import failed. The store files in /xxx/data/databases/example.db are left as they are, although they are likely in an unusable state. Starting a database on these store files will likely fail or observe inconsistent records so start at your own risk or delete the store manually
unexpected error: ERROR in input
  data source: BufferedCharSeeker[source:/xxx/example.csv, position:196, line:1]
  in field: :LABEL:2
  for header: [id:ID, :LABEL]
  raw field value: label_0;label_1;label_2;label_3;label_4;label_5;label_6;label_7;label_8;label_9;label_10;label_11;label_12;label_13;label_14;label_15;label_16;label_17;label_18;label_19;label_20
  original error: null
```

**Comment by tinwelint** [comment]
This is fixed in 3.4 and forwards.

**Comment by tinwelint** [comment]
Thanks, I found the issue and it will be fixed in the next 3.5.x release
